server/routes/updateChainEntityTitle.ts

```diff
@@ -0,0 +1,52 @@
+import { Request, Response, NextFunction } from 'express';
+import { Op } from 'sequelize';
+import proposalIdToEntity from '../util/proposalIdToEntity';
+import lookupCommunityIsVisibleToUser from '../util/lookupCommunityIsVisibleToUser';
+
+export const Errors = {
+  NoEntity: 'Cannot find entity',
+  NotAdminOrOwner: 'Not an admin or owner of this entity',
+};
+
+const updateChainEntityTitle = async (models, req: Request, res: Response, next: NextFunction) => {
+  const [chain, community, error] = await lookupCommunityIsVisibleToUser(models, req.body, req.user);
+  if (error) return next(new Error(error));
+  const { unique_id, title } = req.body;
+
+  const entity = await proposalIdToEntity(models, chain.id, unique_id);
+  if (!entity) return next(new Error(Errors.NoEntity));
+  const userOwnedAddressIds = await req.user.getAddresses().filter((addr) => {
+    return !!addr.verified;
+  }).map((addr) => addr.id);
+  // Todo: author check
```

Comment:
What do you need to complete this TODO?

Reply:
This would be in part 4, we need to save the address that authored the proposal so anyone with that address linked can also set the proposal title.